cswinrt/strings/additions/Windows.Storage.Streams/WindowsRuntimeBufferExtensions.cs

```diff
@@ -118,9 +119,8 @@ public static byte[] ToArray(this IBuffer source)
         public static byte[] ToArray(this IBuffer source, uint sourceIndex, int count)
         {
             if (source == null) throw new ArgumentNullException(nameof(source));
-            if (source.Length == 0 && count == 0) return Array.Empty<byte>();
             if (count < 0) throw new ArgumentOutOfRangeException(nameof(count));
-            if (source.Capacity <= sourceIndex) throw new ArgumentException(global::Windows.Storage.Streams.SR.Argument_BufferIndexExceedsCapacity);
+            if (source.Capacity < sourceIndex) throw new ArgumentException(global::Windows.Storage.Streams.SR.Argument_BufferIndexExceedsCapacity);
```

Comment:
shouldn't these checks be against source.Length, and not source.Capacity?

Reply:
The API documentation suggests Capacity, https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.windowsruntime.windowsruntimebufferextensions.toarray?view=dotnet-plat-ext-3.1 

I realize it also suggests `<=` instead of `<` so I think I need to see about getting the docs updated there.